lib/db.js

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 var P = require('bluebird')
+var extend = require('util')._extend;
```

Comment:
Yeah, from recent chatter re: iojs@2.2.0, it's unlikely that util._extend can ever be removed, and if ultimately removed, it won't be sudden (at least not intentionally).